app/src/main/res/layout/content_send.xml

```diff
@@ -363,7 +364,7 @@
                     android:layout_height="wrap_content"
                     android:fontFamily="@font/source_sans_pro_regular"
                     android:gravity="center"
-                    android:text="@string/fee_colon"
+                    android:text="@string/estimatedFee"
```

Comment:
Looks like you had mistakes in your rebase. fee_colon is the updated code and I see same mistake everywhere.